Applications/Piano/AudioEngine.cpp

```diff
@@ -143,6 +147,37 @@ void AudioEngine::reset()
     m_previous_column = horizontal_notes - 1;
 }
 
+const String AudioEngine::set_clip(const StringView& path)
```

Comment:
The name `clip` is really wrong for this, as you noted in the commit message.
Let's find something better. Some ideas:
* WaveSample
* AudioSample
* SoundSample
* RecordedSample

We could also rename `Audio::Sample` to something else, like `Audio::StereoSample`. I don't know. Let's just not call this `clip` since clipping means something else.